examples/deployments/databricks.py

```diff
@@ -55,36 +55,40 @@ def main():
     )
     try:
```

Comment:
Do we need this try/finally? Can we remove?

Reply:
I want to keep it to remove the endpoint on error.